### ISSUE TITLE
Create an About Page

### ISSUE THREAD

**Issue by BryceV** [cpe308-309/AvengersRepo#52]

**Comment by JPautz** [comment]
Is this the how to page? Like the user manual/help page?

**Comment by BryceV** [comment]
No, this is a page for describing the team basically. In the context of "Poly Path was created by a group of Cal Poly student in 2017...". Maybe throw up a team picture.
